pkg/oc/cli/admin/migrate/scc/scc_to_psp_test.go

```diff
@@ -551,28 +549,28 @@ func TestExtractSeccompProfilesWithWildcardAndNamedProfiles(t *testing.T) {
 	}
 }
 
-func TestExtractSysctls(t *testing.T) {
-	expected := "net.ipv4.route.*,kernel.msg*"
-	sysctlsAnnotation := internalextensions.SysctlsPodSecurityPolicyAnnotationKey
-
-	scc := &securityapi.SecurityContextConstraints{
-		ObjectMeta: metav1.ObjectMeta{
-			Annotations: map[string]string{
-				sysctlsAnnotation: expected,
-			},
-		},
-	}
-
-	annotations := make(map[string]string)
-	extractSysctls(scc, annotations)
-
-	actual, hasAnnotation := annotations[sysctlsAnnotation]
-	if !hasAnnotation {
-		t.Errorf("extractSysctls() expected to set annotation %q but it is not set", sysctlsAnnotation)
-	} else if actual != expected {
-		t.Errorf("extractSysctls() expected annotation %q to be %q but it has value %q", sysctlsAnnotation, expected, actual)
-	}
-}
+// func TestExtractSysctls(t *testing.T) {
+// 	expected := "net.ipv4.route.*,kernel.msg*"
+// 	sysctlsAnnotation := internalextensions.SysctlsPodSecurityPolicyAnnotationKey
+
+// 	scc := &securityapi.SecurityContextConstraints{
+// 		ObjectMeta: metav1.ObjectMeta{
+// 			Annotations: map[string]string{
+// 				sysctlsAnnotation: expected,
+// 			},
+// 		},
+// 	}
+
+// 	annotations := make(map[string]string)
+// 	extractSysctls(scc, annotations)
+
+// 	actual, hasAnnotation := annotations[sysctlsAnnotation]
+// 	if !hasAnnotation {
+// 		t.Errorf("extractSysctls() expected to set annotation %q but it is not set", sysctlsAnnotation)
+// 	} else if actual != expected {
+// 		t.Errorf("extractSysctls() expected annotation %q to be %q but it has value %q", sysctlsAnnotation, expected, actual)
+// 	}
+// }
```

Comment:
Either remove or update a test. I don't see a reason to have the code commented.

Reply:
It must be a relict that I forgot to fix, thank you for catching this.